.github/workflows/azure_automation_test.yml

```diff
@@ -1,19 +1,20 @@
-# if you want to debug this and trigger it on PRs against dev
-# uncomment the 2 pull_request lines below
-
 name: Azure automation tests
+run-name: ${{ inputs.run-name }}
 
 on:
-#  pull_request:
-#    branches: [ develop ]
-  schedule:
```

Comment:
do we not need to run on a schedule anymore? I'm fine with it, just not sure what the QA requirements are

Reply:
Yeah, the scheduling was just to give us regular data on test runs. The QA requirement is running them on the beehive hook from staging, and we will be getting our regular data from the suite run whenever a PR is merged to develop